spec/models/page_spec.rb

```diff
@@ -581,4 +581,61 @@
       expect(page.total_donations.to_s).to eq '1200.0'
     end
   end
+
+  describe '#donation_page?' do
+    before  do
+      @page = create :page # , liquid_layout: donation
+    end
+
+    it 'should return false' do
+      [create(:plugins_petition, page: @page),
+       create(:plugins_fundraiser, page: @page)]
+      expect(@page.donation_page?).to be_falsey
+    end
+
+    it 'should return true' do
+      [create(:plugins_donations_thermometer, page: @page),
+       create(:plugins_fundraiser, page: @page)]
+      expect(@page.donation_page?).to be_truthy
+    end
+
+    it 'should return false' do
+      [create(:call_tool, page: @page)]
+      expect(@page.donation_page?).to be_falsey
```

Comment:
`be_falsey` is the wrong matcher. I think you'll want to use `to be false`, as the method returns a boolean.